renovate.json

```diff
@@ -1,7 +1,17 @@
 {
+  "separateMajorMinor": false,
   "extends": [
-    "config:base"
+    ":preserveSemverRanges",
+    "group:all",
```

Comment:
I think we can just add `:preserveSemverRanges` and `config:base` for the purpose

And monthly schedule sounds nice. But I think it'll try to accommodate all changes in one PR and it'd be hard to figure out which dependency is breaking the build(if it fails)

And IMO the lockFileMaintenance block is redundant.

Please let me know what you think about these points.

Reply:
I think yes monthly updates will be difficult to handle if build is failing. But if we do not pass that option then what will the frequency of updates?

Reply:
They'll come as soon as a new version of a dependency is published.

Reply:
Then I guess I should remove monthly scheduling for update.